SAS/Jonckheere-Terpstra Test.qmd

```diff
@@ -0,0 +1,106 @@
+------------------------------------------------------------------------
+
+title: "SAS Jonckheere-Terpstra Test"
+
+------------------------------------------------------------------------
+
+## Background
+
+The Jonckheere-Terpstra (JT) test is a nonparametric method designed to detect ordered differences across categories. It offers an advantageous alternative to more general tests for class differences, such as the Kruskal-Wallis test, particularly when the analysis is conducted using the WILCOXON option within the NPAR1WAY procedure. $^{[1]}$
+
+The JT test is particularly well-suited for dose-response or trend analysis with ordered categorical data, where the objective is to ascertain whether an increment in dosage leads to a corresponding escalation or reduction in the response variable.$^{[2]}$$^{[5]}$ Unlike other statistical evaluations that might focus on identifying isolated differences between groups, this test is specifically tailored to uncover an overarching trend within the data.
+
+## SAS Procedure
+
+To request Jonckheere-Terpstra test, specify the **JT** option in the Table statement like below:
+
+Preq; table Var1 \* Var2 / **JT** ; Quit;
+
+The JT option in the TABLES statement provides the Jonckheere-Terpstra test.
```

Comment:
Repetition of line 15?

Reply:
> Repetition of line 15?

It's example code, now it's formatted as code block.